src/platform_impl/macos/view.rs

```diff
@@ -670,29 +764,37 @@ extern "C" fn key_down(this: &Object, _sel: Sel, event: id) {
             },
         };
 
-        let pass_along = {
-            AppState::queue_event(EventWrapper::StaticEvent(window_event));
-            // Emit `ReceivedCharacter` for key repeats
-            if is_repeat != NO {
-                for character in characters.chars().filter(|c| !is_corporate_character(*c)) {
-                    AppState::queue_event(EventWrapper::StaticEvent(Event::WindowEvent {
-                        window_id,
-                        event: WindowEvent::ReceivedCharacter(character),
-                    }));
+        // When you operate IME, you should avoid these keycode because they make side effect in window.
+        match virtual_keycode.unwrap() {
+            VirtualKeyCode::Back
+            | VirtualKeyCode::Up
+            | VirtualKeyCode::Down
+            | VirtualKeyCode::Left
+            | VirtualKeyCode::Right => {}
+            _ => {
+                AppState::queue_event(EventWrapper::StaticEvent(window_event));
+            }
+        };
+
+        // Emit `ReceivedCharacter` for key repeats
+        if is_repeat == YES {
```

Comment:
This is not guaranteed to work. You should compare the `BOOL` with `NO` or compare it with 0.
```suggestion
        if is_repeat != NO {
```

Reply:
I'm not inexperienced on macOS, and I thought this would return `YES`. 
I fixed. Thanks.